shared/markdown/parser.pegjs

```diff
@@ -73,7 +73,7 @@ Text
 
 // TODO: should coalesce multiple line quotes
```

Comment:
Note: while the styling of adjacent quote blocks makes them looks like a continuous line, so this is visually correct, we should parse adjacent lines of quotes like...

```
> foo
> bar
```

...as a single quote containing a newline rather than two quotes. We can defer for now, leaving this TODO.